pyinfra/modules/yum.py

```diff
@@ -143,6 +143,7 @@ def update(state, host):
 def packages(
     state, host, packages=None,
     present=True, latest=False, update=False, clean=False,
+    installArgs='', uninstallArgs='',
```

Comment:
For consistency could we call these `extra_install_args` and `extra_uninstall_args`?

Reply:
I was actually impressed that you implemented so many options for all of the operations.

Yes, I am lazy and wanted to use a "catch-all".

There is a down-side risk to this approach. If a user wanted to inject some nefarious things in the string, they could.

I only added the "force" because I went looking for a flag that was not yet implemented.